lib/ood_core/batch_connect/templates/vnc.rb

```diff
@@ -86,8 +86,13 @@ def before_script
                 # Clean up any old VNC sessions that weren't cleaned before
                 #{vnc_clean}
 
+                #for turbovnc 3.0 compatability.
+                if timeout 2 vncserver --help 2>&1 | grep 'nohttpd' >/dev/null 2>&1; then
```

Comment:
Sure be nice if there was a way to get the version, but just looked and appears the module installed on OSC systems does not have any kind of version flag.

Reply:
Lol yea that's why I went for this version. Also may be compatible with `tigervnc`? I've seen folks have that installed instead and it cause issues.

Reply:
We install tigervnc on bastion servers:

```
$ vncserver --help

usage: vncserver [:<number>] [-name <desktop-name>] [-depth <depth>]
                 [-geometry <width>x<height>]
                 [-pixelformat rgbNNN|bgrNNN]
                 [-fp <font-path>]
                 [-cc <visual>]
                 [-fg]
                 [-autokill]
                 [-noxstartup]
                 [-xstartup <file>]
                 [-fallbacktofreeport]
                 <Xvnc-options>...

       vncserver -kill <X-display>

       vncserver -list
```

Looks like your logic would work there too.